datacenter/dtr/2.2/guides/admin/configure/job-runner.md

```diff
@@ -0,0 +1,192 @@
+all of the `nautilus_scan_check_single` jobs are complete, this job will terminate.
+- `nautilus_scan_check_single`
+: A security scanning job for a particular layer given by the `parameter: SHA256SUM`. This job
+breaks up the layer into components and checks each component for vulnerabilities
+(see [Security Scanning](../../user/manage-images/scan-images-for-vulnerabilities.md)).
+- `nautilus_update_db`
+: A job that is created to update DTR's vulnerability database. It uses an
+Internet connection to check for database updates through `https://dss-cve-updates.docker.com/` and
+updates the dtr-scanningstore container if there is a new update available (see [Set up vulnerability scans](set-up-vulnerability-scans.md)). 
+- `webhook`
+
+#### Job Capacities
+As mentioned above, each jobrunner container acts as one worker that can carry out these actions.
+The number of a particular action a worker can carry out is defined by it's capacity which can be 
+seen in the `GET /api/v0/workers` endpoint. For example the workers entry may look like this:
+
+```json
+{
+  "workers": [
+    {
+      "id": "000000000000",	
+      "status": "running",
+      "capacityMap": {
+        "scan": 1,
+        "scanCheck": 1
+      },
+      "heartbeatExpiration": "2017-02-18T00:51:02Z"
+    }
+  ]
+}
+```
+
+This means that the worker with the replica ID `000000000000` has a capacity of 1 for `scan` jobs and 1
+for `scanCheck` jobs. A job may have a `capacityMap` field which dictates how much capacity a worker
+must have available for the job to be executed.
+
+For example, if we take the above worker's `capacityMap` and the following jobs:
+
+```json
+{
+  "jobs": [
+    {
+      "id": "0",
+      "workerID": "",
+      "status": "waiting",
+      "capacityMap": {
+        "scan": 1
+      }
+    },
+    {
+       "id": "1",
+       "workerID": "",
+       "status": "waiting",
+       "capacityMap": {
+         "scan": 1
+       }
+    },
+    {
+     "id": "2",
+      "workerID": "",
+      "status": "waiting",
+      "capacityMap": {
+        "scanCheck": 1
+      }
+    }
+  ]
+}
+```
+
+Our worker will be able to pick up job id `0` and `2` since it has the capacity for both,
+while id `1` will have to wait until the previous scan job is complete:
+
+```json
+{
+  "jobs": [
+    {
+      "id": "0",
+      "workerID": "000000000000",
+      "status": "running",
+      "capacityMap": {
+        "scan": 1
+      }
+    },
+    {
+       "id": "1",
+       "workerID": "",
+       "status": "waiting",
+       "capacityMap": {
+         "scan": 1
+       }
+    },
+    {
+     "id": "2",
+      "workerID": "000000000000",
+      "status": "running",
+      "capacityMap": {
+        "scanCheck": 1
+      }
+    }
+  ]
+}
+```
+
+## Job Statuses
+
+Jobs can have the following statuses:
+
+- `waiting`
+- `running`
+- `done`
+- `error`
+- `cancel_request`
+- `cancel`
+- `deleted`
+- `timeout`
+- `worker_dead`
+- `worker_shutdown`
```

Comment:
worker_shutdown happens whenever a worker is gracefully stopped with docker stop